### PR TITLE
Update of Item Tree for 1.9pr3

### PR DESCRIPTION
**Pull request by Larandar**
Missing : Lot of different potions.

### CODE REVIEW

**Comment by ghost** [comment]
Thanks. In my PM I didn't think about the `<potion id="373"/>` workaround, but indeed it should work as expected. Have you tested it already?

File: src/DefaultTree.dat [hunk 1]
```diff
@@ -451,7 +456,31 @@
 			<melonStem id="105"/>
 			<lilyPad id="111"/>
 		</plant>
+		<potions>
```
Comment (Larandar) [inline src/DefaultTree.dat:459]: The problem is that a unknow number of different potions can be existing, maybe this node have to be named as "potion" and the "potion" node rename to "genericPotion" ??

File: src/DefaultTree.dat [hunk 2]
```diff
@@ -451,7 +456,31 @@
...
+			<potionFireResist>
+				<potionFireResist8 id="373" damage="67"/>
+				<potionFireresist3 id="373" damage="35"/>
+				<potionFireResist3 id="373" damage="3"/>
+			</potionFireResist>
+			<noEffects>
+				<potionAwkward id="373" damage="32"/>
+				<potionThick id="373" damaga="16"/>
+			</noEffecst>
+			<potion id="373"/>
```
Comment (Larandar) [inline src/DefaultTree.dat:474]: Here to match all potion, as write in line 459 maybe rename it to "genericPotion" ??

File: src/DefaultTree.dat (hunk 1 above)
Comment (ghost) [inline src/DefaultTree.dat:459]: Indeed the category and the item are a bit redundant, I just renamed the item to "anyPotion".

**Pull request merged**